SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Astro Security
+## Reporting a Vulnerability
```

Comment:
Would you mind submitting this file to [https://github.com/withastro/.github](https://github.com/withastro/.github) instead? I just added that repo so things like this can be shared for every repo in our organization.

Reply:
I took care of this! https://github.com/withastro/.github/blob/main/SECURITY.md